CONTRIBUTING.md

```diff
@@ -50,6 +50,10 @@ Cobalt coding style.
 
 #### General guidelines and philosophy for contribution
 
+*   Ensure your change references the bug you are addressing. Follow the
```

Comment:
Should we add the `references the bug in form b/124 on a separate commit line`, for now ? 

Obviously i want to make it easier and just require it in PR description soon, but that work still needs doing.

Reply:
Can we just do that work? Isn't there already a PR with it?

Reply:
No there isn't ye. We'd need to flip the checks over to verify only PR title and description, and ensure it works for all cases. I'm also expecting it to break gitwatcher, so we'd need an alternative.

Reply:
This is all that's needed https://github.com/youtube/cobalt/pull/408
What do you expect to break with gitwatcher? The way squashed + merged commits are migrated won't change